src/components/orders/order-form/OrderSideTab.tsx

```diff
@@ -0,0 +1,44 @@
+import { Tabs, TabsList, TabsTrigger } from "@/components/ui/tabs.tsx";
+import type { FC } from "react";
+
+type OrderSideTabProps = {
+  orderFormType: string;
+  onOrderFormTypeChange: (orderFormType: string) => void;
+};
+
+const OrderSideTab: FC<OrderSideTabProps> = ({
+  orderFormType,
+  onOrderFormTypeChange,
+}) => {
+  return (
+    <Tabs
+      value={orderFormType}
+      onValueChange={(value: string) => {
+        onOrderFormTypeChange(value);
+      }}
```

Comment:
onValueChange에서 받는 함수의 매개변수 타입이 전달하는 함수의 매개변수 타입과 같다면, 그냥 아래와 같이 쓰는게 좋습니다.

```suggestion
      onValueChange={onOrderFormTypeChange}
```

이전 코드처럼 하면 매 렌더링 때마다 onValueChange에 들어가는 함수가 새로 생성되는데, 이렇게 하면 onOrderFormTypeChange의 참조가 바뀌지 않는 한 그 오버헤드를 제거할 수 있습니다.